packages/app/app/containers/SoundContainer/index.js

```diff
@@ -183,14 +183,18 @@ class SoundContainer extends React.Component {
   }
 
   isHlsStream(url) {
-    return /http.*?\.m3u8/g.test(url);
+    if (url?.stream){
+      return url.stream?.includes('m3u8');
+    } else {
```

Comment:
Not sure if HLS player is currently working as expected. When files are played there is no progress bar. 

Also the HLS stream detecting method only supports an array of streams.

Reply:
HLS is for live streams, so there's no progress bar, because there's no way to tell when it's going to end.